framework/utils/mathUtils.py

```diff
@@ -536,14 +536,16 @@ def numBinsDraconis(data, low=None, alternateOkay=True):
       numBins = int(np.ceil(np.sqrt(data.size)))
     else:
       raise ValueError('When computing bins using Freedman-Diaconis the 25th and 75th percentiles are the same, and "alternate" is not enabled!')
+  if binOps == 2:
```

Comment:
if binOps == 2, why do we compute the Draconis rule before if we will overwrite it no matter what?

Reply:
Good point. Fixed.